app/src/androidTest/java/com/mobiledevelopmentfoodapp/ExampleInstrumentedTest.kt

```diff
@@ -8,17 +8,14 @@ import org.junit.runner.RunWith
 
 import org.junit.Assert.*
 
-/**
- * Instrumented test, which will execute on an Android device.
- *
- * See [testing documentation](http://d.android.com/tools/testing).
- */
+/** Instrumented test, which will execute on an Android device. */
 @RunWith(AndroidJUnit4::class)
 class ExampleInstrumentedTest {
     @Test
     fun useAppContext() {
         // Context of the app under test.
-        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        val appContext =
+            InstrumentationRegistry.getInstrumentation().targetContext
```

Comment:
Changed the formatting of this initializer according to https://kotlinlang.org/docs/coding-conventions.html#properties